autoload/lsp/util.vim

```diff
@@ -133,6 +138,31 @@ export def LspFileToUri(fname: string): string
   return uri
 enddef
 
+# Convert POSIX paths as used in Cygwin to native Windows paths
+def CygwinToWindowsPath(path: string): string
```

Comment:
there should be direct way to convert convert such path in cygwin, if i didnot recall wrong.
// so most code of this func perhaps is not required.

// and other than this, not sure lsp server returned path was window or unix style path,
// somewhere perhaps should convert it oppositely.

Reply:
> there should be direct way to convert convert such path in cygwin, if i didnot recall wrong.

Hm, you seem to be right, there is [`cygpath`](https://cygwin.com/cygwin-ug-net/cygpath.html). I'll try to replace my code with it.

Reply:
I have merged this PR.  Can you create a new PR with the change?